tests/integration/acp/dac/add_policy/with_no_perms_test.go

```diff
@@ -32,22 +32,11 @@ func TestACP_AddPolicy_NoPermissionsOnlyOwner_ValidID(t *testing.T) {
 				Identity: testUtils.ClientIdentity(1),
 
 				Policy: `
-                    name: test
-                    description: a policy
-
-                    actor:
-                      name: actor
-
-                    resources:
-                      users:
-                        permissions:
-
-                        relations:
-                          owner:
-                            types:
-                              - actor
-
-                `,
+description: a policy
+name: test
+resources:
+- name: users
+`,
```

Comment:
todo: This test is now a duplicate of `TestACP_AddPolicy_NoPermissionsLabelOnlyOwner_ValidID`. This test was previously testing the equivalent case below:

```suggestion
description: a policy
name: test
resources:
- name: users
  permissons:
`,
```